easybuild/easyconfigs/c/Cuby/Cuby-4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = "PackedBinary"
+
+name = "Cuby"
+version = '4'
+
+homepage = 'http://cuby.molecular.cz/cuby4/'
+description = """Cuby is a computational chemistry framework written in ruby.
+ For users, it provides an unified access to various computational
+ methods available in difefrent software packages. For developers,
+ Cuby is much more - it is a complex framework that provides
+ object-oriented access to the data enetering the calculations and
+ to their results, making it easy to create new computational
+ protocols by combining existing blocks of the framework."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
+1

I would even go as far as using a non-dummy toolchain here, since you're picking a toolchain anyway when specifying the dependency

that'll make it clear that this module will also result in loaded `intel/2014b`